pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[tool.poetry]
+name = "MM-classification-service"
+version = "0.1.0"
+description = ""
+authors = []
```

Comment:
You seem to be missing package information

Reply:
As mentioned before, I plan on updating package and dependancy information once Jason onboards the ML side of our classification service.